packages/create-schemas/tests/watch.test.ts

```diff
@@ -7,7 +7,8 @@ import type { GenerationResult } from "../src/generate.ts";
 import { openapiTypeScriptFilename } from "../src/plugins/openapi-typescript-plugin.ts";
 
 describe("watch", () => {
-    test("changing the input", async ({ expect, onTestFinished }) => {
+    // TODO: Test is flaky... doesn't trigger rebuilds consistently
```

Comment:
Remaining TODO

Reply:
I added to the TODO since the test was flaky. I'll remove it in another PR.